documentation/src/main/asciidoc/topics/proc_xsite_monitoring.adoc

```diff
@@ -0,0 +1,67 @@
+[id='monitor-xsite-replication_{context}']
+= Monitoring the Cross-Site Replication
+
+After enabling cross-site replication with your {brandname} server, cross-site replication status should be monitored to detect interruptions in the replication.
+When a remote site changes status to `offline`, {brandname} stops replicating your data to that specific site.
+Your data becomes out of sync between your sites.
+
+Monitoring these events is necessary.
+Admin intervention is required to fix the inconsistencies in your data and to re-enable cross-site replication.
+
+Below are provided two alternatives how the cross-site replication can be monitored.
+The first one uses the REST endpoint and the second uses the exposed metrics.
+These metrics can be scrapped by Prometheus or any other monitoring software compatible.
+
+== Using the REST endpoint
+
+Polling the REST endpoint, it is possible to obtain the site status for all caches.
+
+The example below show a python script that poll the site status every 5 seconds.
+It is a good start how to monitor the cross-site statuses.
+The global variables are:
+
+* `USERNAME` and `PASSWORD`: The username and password of a {brandname} user with access to the REST endpoint.
+* `POLL_INTERVAL_SEC`: The number of seconds between polls.
+* `SERVERS`: The list of {brandname} servers.
+The script only requires a single valid response but the list is provided to allow fail over.
+* `REMOTE_SITES`: The list of remote sites to monitor on these servers.
+* `CACHES`: The list of cache names to monitor.
+
```

Comment:
Is it possible to let SITES and CACHES empty to monitor all which have XSite config?

Reply:
It is not possible with the current python code, but I can change it.

Reply:
I take that back :rofl: It adds too much complexity IMO. I would have to fetch all cache configurations at every interval to check the cross-site configuration.

Reply:
So I would mention that SITES and CACHES are mandatory

Reply:
All of them are mandatory.

Reply:
I would think user/password is optional if there is no security